cmd/auth.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/prest/prest/adapters/postgres"
+	"github.com/prest/prest/config"
+	"github.com/spf13/cobra"
+)
+
+var authUpCmd = &cobra.Command{
```

Comment:
command not show in `--help`
```
❯ prestd (avelino/auth) ✘ go run main.go --help
Serve a RESTful API from any PostgreSQL database, start HTTP server

Usage:
  prest [flags]
  prest [command]

Available Commands:
  help        Help about any command
  migrate     Execute migration operations
  version     Print the version number of pREST

Flags:
  -h, --help   help for prest

Use "prest [command] --help" for more information about a command.
```

Reply:
It's available inside migrate up.
Do you think will be better if the command was ```auth up``` and ``` auth down```?

Reply:
no, but we need to document
```
go run main.go migrate up auth
go run main.go migrate down auth
```